app/src/main/java/com/github/se/orator/ui/settings/SettingsScreen.kt

```diff
@@ -45,66 +43,22 @@ import com.github.se.orator.ui.theme.AppFontSizes
 data class SettingBar(
     val text: String,
     val testTag: String,
-    val function: (AppThemeViewModel?) -> Unit,
     val icon: ImageVector,
     val iconDescription: String
 )
 
 // creating a list of all settings to implement them with a simple loop
 val listOfSettings =
     listOf(
-        SettingBar(
-            "Account Management",
-            "account_management",
-            { Log.d("hello", "account management") },
-            Icons.Outlined.AccountCircle,
-            "account circle icon"),
-        SettingBar(
-            "Storage Settings",
-            "storage_settings",
-            { Log.d("hello", "storage settings") },
-            Icons.AutoMirrored.Outlined.List,
-            "storage settings icon"),
-        SettingBar(
-            "Permissions",
-            "permissions",
-            { Log.d("hello", "permissions") },
-            Icons.Outlined.Lock,
-            "lock icon"),
-        SettingBar(
-            "Theme (click to switch)",
-            "theme",
-            { themeVM ->
-              themeVM?.switchTheme()
-              Log.d("hello", "theme")
-            },
-            Icons.Outlined.DarkMode,
-            "theme"),
-        SettingBar(
-            "Invite Friends",
-            "invite_friends",
-            { Log.d("hello", "invite friends") },
-            Icons.Outlined.Mail,
-            "mail icon"),
-        SettingBar(
-            "Notifications",
-            "notifications",
-            { Log.d("hello", "Notifications") },
-            Icons.Outlined.Notifications,
-            "notifications icon"),
-        SettingBar(
-            "Rate on the App Store",
-            "rate_on_the_app_store",
-            { Log.d("hello", "rate on the app store") },
-            Icons.Outlined.Star,
-            "star icon"),
-        SettingBar("About", "about", { Log.d("hello", "about") }, Icons.Outlined.Info, "info icon"))
+        SettingBar("Permissions", "permissions", Icons.Outlined.Lock, "lock icon"),
+        SettingBar("Theme", "theme", Icons.Outlined.DarkMode, "theme"),
+        SettingBar("About", "about", Icons.Outlined.Info, "info icon"))
 
```

Comment:
I would suggest adding brief comments in the code to indicate the purpose of the retained buttons ("permissions," "theme," and "about"). For example, note that the "about" button is reserved for a future feature.

Reply:
Hey, I followed your suggestion and also removed the unused "about" button. Let me just stress something though : I never removed any previous documentation, nor added any new button. There already were 8~9 buttons unused and undocumented.